{
  "schema_version": "1.4.0",
  "id": "GHSA-c33m-px5c-j98p",
  "modified": "2025-01-08T00:30:47Z",
  "published": "2024-12-29T12:30:40Z",
  "aliases": [
    "CVE-2024-56728"
  ],
  "details": "In the Linux kernel, the following vulnerability has been resolved:\n\nocteontx2-pf: handle otx2_mbox_get_rsp errors in otx2_ethtool.c\n\nAdd error pointer check after calling otx2_mbox_get_rsp().",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-56728"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/05a6ce174c0c724e5914e1e5efd826bab8f382b4"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/2db2194727b1f49a5096c1c3981adef1b7638733"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/55c41b97001a09bb490ffa2e667e251d75d15ab1"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/5ff9de1f2712cbca53da2e37d831eea7ffcb43b6"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/6cda142cee032b8fe65ee11f78721721c3988feb"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/c0f64fd73b60aee85f88c270c9d714ead27a7b7a"
    },
    {
      "type": "WEB",
      "url": "https://git.kernel.org/stable/c/e26f8eac6bb20b20fdb8f7dc695711ebce4c7c5c"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-754"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2024-12-29T12:15:06Z"
  }
}